sdk/storage/storage-blob-changefeed/test/utils/testutils.common.ts

```diff
@@ -148,12 +148,11 @@ export function sleep(seconds: number): Promise<void> {
 }
 /**
  * Read text content from a stream
- * @param stream
+ * @param stream -
```

Comment:
if there's no description, maybe just delete this line?

Reply:
I added some descriptions